catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java

```diff
@@ -96,31 +96,31 @@
         descriptors.add(new AttributeDescriptorImpl(DERIVED_RESOURCE_DOWNLOAD_URL,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
+                false /* tokenized */,
                 true /* multivalued */,
                 BasicTypes.STRING_TYPE));
         descriptors.add(new AttributeDescriptorImpl(DERIVED_RESOURCE_URI,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
+                false /* tokenized */,
                 true /* multivalued */,
                 BasicTypes.STRING_TYPE));
         descriptors.add(new AttributeDescriptorImpl(RESOURCE_DOWNLOAD_URL,
-                true /* indexed */,
+                false /* indexed */,
                 false /* stored */,
-                true /* tokenized */,
+                false /* tokenized */,
                 false /* multivalued */,
                 BasicTypes.STRING_TYPE));
         descriptors.add(new AttributeDescriptorImpl(RESOURCE_SIZE,
-                true /* indexed */,
+                false /* indexed */,
```

Comment:
Javadoc says this is the size in bytes. If that is true, it should be a numeric type and indexed so that users can perform range queries.